ams/server/src/main/java/com/netease/arctic/server/DefaultOptimizingService.java

```diff
@@ -19,6 +19,8 @@
 package com.netease.arctic.server;
 
 import com.google.common.base.Preconditions;
+import com.google.common.collect.Maps;
+import com.google.common.util.concurrent.ThreadFactoryBuilder;
```

Comment:
Please use shaded guava class. 
```suggestion
import org.apache.iceberg.relocated.com.google.common.base.Preconditions;
import org.apache.iceberg.relocated.com.google.common.collect.Maps;
import org.apache.iceberg.relocated.com.google.common.util.concurrent.ThreadFactoryBuilder;
```